MatrixKit/Views/Authentication/MXKAuthenticationFallbackWebView.m

```diff
@@ -67,6 +76,15 @@ - (void)openFallbackPage:(NSString *)fallbackPage success:(void (^)(MXCredential
     activityIndicator.center = self.center;
     [self addSubview:activityIndicator];
     [activityIndicator startAnimating];
+
+    // Delete CAS OM cookies to prevent auto login
+    for(NSHTTPCookie *cookie in [[NSHTTPCookieStorage sharedHTTPCookieStorage] cookies])
+    {
+        if([[cookie domain] isEqualToString:@"cas.openmarket.com"])
```

Comment:
I am not comfortable with putting a special handling on a particular page url. 
Can we clean cookies every time?